src/agent/state/inspector-state.ts

```diff
@@ -284,18 +285,26 @@ class StateResolver {
     return this.stripCurrentWorkingDirectory_(fullPath);
   }
 
+  stripFileProtocol_(path: string) {
```

Comment:
nit: make this a static function. Ideally `isPathInNodeModulesDirectory_` would be static too but that can happen in a different PR.